auto_gptq/utils/import_utils.py

```diff
@@ -66,6 +66,14 @@ def dynamically_import_QuantLinear(
     use_qigen: bool = False,
     disable_marlin: bool = True,
 ):
+    # if use_hpu:
+    try:
+        import habana_frameworks.torch.hpu  # noqa: F401
+    except ImportError as e:
+        pass
+    else:
```

Comment:
else of what?

Reply:
Of the try-except

The else block lets you execute code when there is no error.

https://www.w3schools.com/python/python_try_except.asp